go-controller/pkg/ovn/namespace.go

```diff
@@ -104,7 +104,7 @@ func (oc *DefaultNetworkController) getRoutingExternalGWs(nsInfo *namespaceInfo)
 	// return a copy of the object so it can be handled without the
 	// namespace locked
 	res.bfdEnabled = nsInfo.routingExternalGWs.bfdEnabled
-	res.gws = sets.New[string](nsInfo.routingExternalGWs.gws.UnsortedList()...)
+	res.gws = sets.New(nsInfo.routingExternalGWs.gws.UnsortedList()...)
```

Comment:
curiosity question: this list is of type string right? so was it done wrongly when we do `New[string]` ?

Reply:
yes you no longer need to pass in the type, it will be inferred from the data